src/main/java/io/axoniq/axonserver/connector/event/impl/BufferedAggregateEventStream.java

```diff
@@ -51,11 +51,23 @@ public BufferedAggregateEventStream() {
 
     /**
      * Constructs a {@link BufferedAggregateEventStream} with the given {@code bufferSize}.
+     * <p>
+     * The permits will be replenished using batches of 1/8th of the given {@code bufferSize}, unless the buffer is
+     * small than 8 items, in which case permits are replenished individually, as items are consumed from the buffer.
```

Comment:
```suggestion
     * smaller than 8 items, in which case permits are replenished individually, as items are consumed from the buffer.
```

Reply:
Good idea. I removed this constructor altogether and added a no-param constructor which uses sensible default values.